models/stream_input_response.go

```diff
@@ -0,0 +1,47 @@
+package models
+
+import "github.com/bitmovin/bitmovin-go/bitmovintypes"
+
+type StreamInputData struct {
+	Result StreamInputResult `json:"result"`
+}
+
+type StreamInputResult struct {
+	FormatName   *string                `json:"formatName"`
+	StartTime    *float64               `json:"startTime"`
+	Duration     *float64               `json:"duration"`
+	Size         *int64                 `json:"size"`
+	Bitrate      *int64                 `json:"bitrate"`
+	AudioStreams []StreamInputAudio     `json:"audioStreams"`
+	VideoStreams []StreamInputVideo     `json:"videoStreams"`
+	Tags         map[string]interface{} `json:"tags"`
+}
+
+type StreamInputAudio struct {
+	ID              *string  `json:"id"`
+	Position        *int64   `json:"position"`
+	Duration        *float64 `json:"duration"`
+	Codec           *string  `json:"codec"`
+	SampleRate      *int64   `json:"sampleRate"`
+	Bitrate         *int64   `json:"bitrate,string"`
```

Comment:
currently in the other models we aren't json string encoding the bitrates.  I will take this under advisement though, as it may make sense to do.  If that's the case, I'll update the entire package with this.

Reply:
Resolved.  It's currently a string.